tests/src/test/java/io/smallrye/context/test/ManualPropagationMultipleRequestTest.java

```diff
@@ -23,10 +23,11 @@
 
     @BeforeClass
     public static void init() {
-        SmallRyeContextManagerProvider.getManager();
         threadContext = (SmallRyeThreadContext) ContextManagerProvider.instance().getContextManager().newThreadContextBuilder()
                 .build();
-        minimalThreadContext = (SmallRyeThreadContext) ContextManagerProvider.instance().getContextManager()
+        minimalThreadContext = SmallRyeContextManagerProvider.instance().getContextManagerBuilder()
+                .withThreadContextProviders(new MyThreadContextProvider()).enableFastThreadContextProviders(true)
```

Comment:
Why do you add that? It's the default, no?

Reply:
Nope, it's not: https://github.com/smallrye/smallrye-context-propagation/blob/master/core/src/main/java/io/smallrye/context/SmallRyeContextManager.java#L256

Reply:
Can we even have it as default? My understanding was that it wouldn't fully align with TCKs or am I mistaken?

Reply:
We could have it as the default (unless I have seriously misread the code), because it will only activate if all providers are instances of `FastThreadContextProvider`

Reply:
DAMNIT. That's my bug, I intended to enable it by default. It won't kick in unless all the `ThreadContextProvider` support it anway, so the TCK will never use that path.

Reply:
OK, I'll enable it

Reply:
Done

Reply:
Ah that makes sense, thanks for explanation!